apis/Google.Cloud.Iam.V1/Google.Cloud.Iam.V1/PolicyPartial.cs

```diff
@@ -26,14 +26,21 @@ public partial class Policy
         /// <summary>
         /// Adds the specified member to the specified role. If the role does
         /// not already exist, it is created.
+        /// This method will fail with an <see cref="InvalidOperationException"/>
+        /// if it is called on a Policy with a <see cref="Version"/> greater than 1,
+        /// or if any of the bindings contain conditions,
+        /// as that indicates a more complicated policy than this method is prepared
+        /// to handle. Changes to such policies must be made manually.
```

Comment:
What does manually mean here, like through the UI?

Reply:
Discussed offline.